src/main/java/org/snomed/snowstorm/core/data/domain/AnnotationView.java

```diff
@@ -0,0 +1,30 @@
+package org.snomed.snowstorm.core.data.domain;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.snomed.snowstorm.core.pojo.TermLangPojo;
+
+@JsonDeserialize(as = Annotation.class)
+public interface AnnotationView {
```

Comment:
@QuyenLy87, the create Concept response is different from the read Concept response in that some properties of the Annotation are only returned when reading the Concept. If these properties are needed by the UI after creating the Annotation, a follow up read request will be needed.

Reply:
Hi @dmcgihtsdo, what do you mean here?

Reply:
Thanks @dmcgihtsdo. I missed some pieces of code and just corrected them. Please have a look. Thanks